ansible/roles/hoover/templates/search-local.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import Path
+base_dir = Path(__file__).absolute().parent.parent.parent.parent
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'hoover-search',
+    },
+}
+
+INSTALLED_APPS = (
+    'hoover.contrib.oauth2',
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'hoover.search',
+)
+
+ALLOWED_HOSTS = ['hoover.{{liquid_domain}}']
+
+STATIC_ROOT = str(base_dir / 'static')
+HOOVER_UPLOADS_ROOT = str(base_dir / 'uploads')
+HOOVER_ELASTICSEARCH_URL = 'http://localhost:14352'
+HOOVER_UI_ROOT = '/opt/hoover/ui/build'
+
+HOOVER_OAUTH_LIQUID_URL = "http{% if use_https %}s{% endif %}://{{ liquid_domain }}"
```

Comment:
This is Jinja, you can define a variable:

```jinja
{% set http_scheme = 'https' if use_https else 'http' -%}
HOOVER_OAUTH_LIQUID_URL = "{{ http_scheme }}://{{ liquid_domain }}"
```

Or maybe set an ansible variable, `http_scheme`, along with `use_https`.